bi_sql_editor_aggregate/models/ir_model.py

```diff
@@ -0,0 +1,14 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo import models
+
+
+class IrModelFields(models.Model):
+    _inherit = 'ir.model.fields'
+
+    def _add_manual_fields(self, model):
+        super()._add_manual_fields(model)
+        if 'bi.sql.view' in self.env:
+            Sql = self.env['bi.sql.view']
+            if hasattr(Sql, 'check_manual_fields'):
```

Comment:
why this check ? 
I guess that it is allways true, if the module is installed ? or did I missed something ?

Reply:
@legalsylvain  This check MAY be unnecessary now, but see my comment below....